### DISCUSSION TITLE
New Firefly install not working

### DISCUSSION THREAD

**Discussion by cmerchant0716** [firefly-iii/firefly-iii#4462] [Firefly III]
I just started to install Firefly and it just states connection timeout in the log. I am attempting to run this on Synology Docker.

I have installed the phpAdmin page and created a database.

![image](https://user-images.githubusercontent.com/72221079/109662608-dbba5300-7b30-11eb-8634-050964985ad7.png)

![image](https://user-images.githubusercontent.com/72221079/109662910-2b008380-7b31-11eb-99fe-5e7b725c2256.png)

![image](https://user-images.githubusercontent.com/72221079/109663073-52575080-7b31-11eb-8290-84d5ea05e7a7.png)

Let me know if any more information is needed.

**Comment by ghost** [comment]
He @cmerchant0716 , a couple of things.

- Make sure you are on php 7.4 or later

- Not sure which db you are using (most likely MySQL).  Try the command line version using the mysql cli.
`CREATE DATABASE firefly;`
`CREATE USER 'firefly'@'localhost' IDENTIFIED BY 'password';`(make sure this matches inside you .env file).

- Change your DB_HOST to **localhost**

**Reply by ghost** [reply]
@cmerchant0716 Any luck?
